### ISSUE TITLE
Mistake in examples\3_NeuralNetworks\convolutional_network.py about "dropout"

### ISSUE THREAD

**Issue by PaulYangSz** [aymericdamien/TensorFlow-Examples#205]
In convolutional_network.py file, line 29 define: `dropout = 0.75 # Dropout, probability to keep units`
And then use in line 60: `fc1 = tf.layers.dropout(fc1, rate=dropout, training=is_training)`
But where `dropout` in function **tf.layers.dropout** is represent the value of parameter `rate`, for convenience I also paste the comment: `rate: The dropout rate, between 0 and 1. E.g. "rate=0.1" would drop out 10% of input units.`
It seems like a conflict usage here, but the actual final result of `rate=dropout` and `rate=1-dropout` are looks pretty similarity.

**Comment by aymericdamien** [comment]
Thanks for reporting that! It is indeed confusing as `tf.nn.dropout` use keep proba. I will push a fix!
